app.js

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-unused-vars */
+function OpenBar() {
+  const closebtn = document.querySelector('.small-screen');
+  closebtn.style.display = 'flex';
+}
+
+function CloseBar() {
+  const closebtn = document.querySelector('.small-screen');
+  closebtn.style.display = 'none';
+}
+
+const projects = [
+  {
+    name: 'Tonic',
+    description:
+        ' Lorem ipsum dolor sit amet consectetur adipisicing elit. Quas molestias iusto architecto blanditiis evenietimpedit tenetur dolorem, non voluptates atque itaque quaerat praesentium incidunt eligendi nostrum animi harumid commodi.Lorem ipsum dolor sit amet consectetur adipisicing elit. Praesentium corporis blanditiis similique neque illomaiores vel repudiandae odit? Facere error suscipit laborum a deleniti rerum dicta quas est corrupti',
+    image: './image/card-1.svg',
+    technologies: ['Html', 'Css', 'Javascript'],
+  },
+  {
+    name: 'Multi-Post Stories',
+    description:
+        ' Lorem ipsum dolor sit amet consectetur adipisicing elit. Quas molestias iusto architecto blanditiis evenietimpedit tenetur dolorem, non voluptates atque itaque quaerat praesentium incidunt eligendi nostrum animi harumid commodi.Lorem ipsum dolor sit amet consectetur adipisicing elit. Praesentium corporis blanditiis similique neque illomaiores vel repudiandae odit? Facere error suscipit laborum a deleniti rerum dicta quas est corrupti',
+    image: './image/card-2.svg',
+    technologies: ['Html', 'Css', 'Javascript'],
+  },
+  {
+    name: 'Tonic',
+    description:
+        ' Lorem ipsum dolor sit amet consectetur adipisicing elit. Quas molestias iusto architecto blanditiis evenietimpedit tenetur dolorem, non voluptates atque itaque quaerat praesentium incidunt eligendi nostrum animi harumid commodi.Lorem ipsum dolor sit amet consectetur adipisicing elit. Praesentium corporis blanditiis similique neque illomaiores vel repudiandae odit? Facere error suscipit laborum a deleniti rerum dicta quas est corrupti',
+    image: './image/card-3.svg',
+    technologies: ['Html', 'Css', 'Javascript'],
+  },
+  {
+    name: 'Multi-Post Stories',
+    description:
+        'A Phaser3 platform game built as part of the Microverse JavaScript program.',
+    image: './image/card-4.svg',
+    technologies: ['Html', 'Css', 'Javascript'],
+  },
+];

```

Comment:
- [ ] Your popup design is not matched with the Figma design. Kindly fix the issue. :+1: 

      >  You should change the font color for the text (back end dev, 2015).
      >  Also, you need to adjust the margin and padding for the Popup.
      > The Buttons (ex: See LIve, See Source) need an icon to match the Figma design.
   
Note: The popup follows the design for both desktop and mobile versions from the chosen template in Figma (e.g., font, colors, images, text, margins).

**Your design**
:-------------------------:
![popupdesign](https://user-images.githubusercontent.com/70483835/189381153-0bee853c-bbfe-46e4-ac20-f3a083e9c232.png)

**Figma Design**
:-------------------------:
![figmapopup](https://user-images.githubusercontent.com/70483835/189381272-122b0776-3abf-4ee8-a458-b973324fe8df.png)